### DISCUSSION TITLE
Expanding row visualization

### DISCUSSION THREAD

**Discussion by petrvecera** [icflorescu/mantine-datatable#65] [General]
Hi guys

I have seen that expanding row just went as a general release https://icflorescu.github.io/mantine-datatable/examples/expanding-rows 
nice work 👍 

One thing which was missing in the example is some button / visualization that the row is expandable. Are we able to implement something like this ? https://ant.design/components/table/#components-table-demo-expand

<img width="861" alt="image" src="https://user-images.githubusercontent.com/8086995/192508175-4760556b-0594-4cdc-8da9-0942a790064c.png">
Obviously you can code that button into a row. But are we able to get somehow info that the row has been expanded so we can change that button ? For example from + to -. 

Thanks
And awesome work so far ❤️

**Comment by icflorescu** [comment]
At the moment, the only cue for expanding a row is the `pointer` mouse cursor.
But I think you're right, having a more appropriate expansion status visualization would be nice.
Maybe we could have something like `trigger: 'action-button'` on the `rowExpansion` property.

Not exactly trivial to implement though, because it would have to work well with horizontal scrolling and in possible combination with the row selection checkboxes that are sticked to left when scrolling...

I'll keep this in mind and definitely come up with an implementation in a future version.

Thanks!
